foojay-resolver/src/test/kotlin/org/gradle/toolchains/foojay/FoojayApiTest.kt

```diff
@@ -66,8 +77,7 @@ class FoojayApiTest {
 
         assertMatchedDistributions(ADOPTOPENJDK, J9, version, "AOJ OpenJ9")
         assertMatchedDistributions(IBM, J9, version, "Semeru")
-        @Suppress("DEPRECATION")
-        assertMatchedDistributions(IBM_SEMERU, J9, version, "Semeru")
+        IBM_SEMERU_VENDOR?.let { assertMatchedDistributions(it, J9, version, "Semeru") }
```

Comment:
❔ Do I understand correctly that there's no way to request a removed vendor?

Reply:
You can, by using `JvmVendorSpec.matching(String)` but then you have to make sure that text matches how Gradle checks for the vendor and how Foojay exposes it.

Reply:
Yep. I asked because I was thinking whether it should be covered by a test or not. I guess it's fine not to have a test for that in the FJ plugin.

Reply:
Pushed an update, PTAL